src/pages/Login/Login.page.jsx

```diff
@@ -21,13 +26,13 @@ function LoginPage() {
         <div className="form-group">
           <label htmlFor="username">
             <strong>username </strong>
-            <input required type="text" id="username" />
+            <input required type="text" id="username" name="username" />
           </label>
         </div>
         <div className="form-group">
           <label htmlFor="password">
             <strong>password </strong>
-            <input required type="password" id="password" />
+            <input required type="password" id="password" name="username" />
```

Comment:
And also you can change this line:

```suggestion
            <input required type="password" {...bindPassword} />
```